src/cultionet/scripts/cultionet.py

```diff
@@ -1276,7 +1276,9 @@ def train_model(args):
         stochastic_weight_averaging_lr=args.stochastic_weight_averaging_lr,
         stochastic_weight_averaging_start=args.stochastic_weight_averaging_start,
         model_pruning=args.model_pruning,
-        save_batch_val_metrics=args.save_batch_val_metrics
+        save_batch_val_metrics=args.save_batch_val_metrics,
+        skip_train=args.skip_train,
+        refine_and_calibrate=args.refine_and_calibrate
```

Comment:
once this PR merge, will a model trained using previous version (before this change) work?

Reply:
The main model architecture isn't modified in this branch. It has been working and I will re-test after fixing the `lr_scheduler` name conflict.